tutorial/README.md

```diff
@@ -0,0 +1,42 @@
+# Why Helm?
+
+[Helm](https://docs.helm.sh/) is often described as the Kubernetes application package manager. So, what does Helm give you over using Kubernetes directly?
+
+# Objectives
+
+These labs provide an insight on the advantages of using Helm instead of using Kubernetes directly through `kubectl`. By the end of this course, you'll:
+* Understand the core concepts of Helm
+* Understand the advantages of deployment using Helm over Kubernetes directly, looking at:
+  * Application management
+  * Updates
+  * Configuration
+  * Revision management
+  * Repositories and chart sharing
+
+# Prerequesites
+
+* Have a running Kubernetes cluster. See the [IBM Cloud Kubernetes Service](https://console.bluemix.net/docs/containers/cs_tutorials.html) or [Kubernetes Getting Started Guide](https://kubernetes.io/docs/setup/) for details about creating a cluster.
+* Have Helm installed and initialized with the Kubernetes cluster. See [Installing Helm on IBM Cloud Kubernetes Service](Lab0/README.md) or the [Helm Quickstart Guide](https://docs.helm.sh/using_helm/#quickstart) for getting started with Helm.
+
+# Helm Overview
+
+Helm is a tool that streamlines installation and management of Kubernetes applications. It uses a packaging format called charts which are a collection of files that describe Kubernetes resources. It can run anywhere (laptop, CI/CD etc.) and is available for various operating systems, like OSX, Linux and Windows. 
```

Comment:
``` Helm is a tool that streamlines installation and management of Kubernetes applications. It uses a packaging format called "charts," which are a collection of files that describe Kubernetes resources. It can run anywhere (laptop, CI/CD, etc.) and is available for various operating systems, like OSX, Linux and Windows. ```

Reply:
done